website/docs/r/eventgrid_system_topic_event_subscription.html.markdown

```diff
@@ -97,6 +97,8 @@ The following arguments are supported:
 
 * `labels` - (Optional) A list of labels to assign to the event subscription.
 
+* `advanced_filtering_on_arrays_enabled` - (Optional) Boolean flag to allows advanced filters to be evaluated against an array of values instead of expecting a singular value. Defaults to `false`.
```

Comment:
```suggestion
* `advanced_filtering_on_arrays_enabled` - (Optional) Specifies whether advanced filters should be evaluated against an array of values instead of expecting a singular value. Defaults to `false`.
```